src/http-data-source.ts

```diff
@@ -325,9 +326,8 @@ export abstract class HTTPDataSource<TContext = any> extends DataSource {
           })
           .catch((err) => this.logger?.error(err))
       }
-
       return response
-    } catch (error) {
+    } catch (error: any) {
```

Comment:
Why is it necessary?

Reply:
my typescript remains me should set type for error, and my version is v4.4.2, but we can keep it empty in lower version.

Reply:
I'm fine with that.